playbooks/satellite/docker-tierdown.yaml

```diff
@@ -37,7 +37,7 @@
     - name: "Start docker daemon"
       service:
         name: docker
-        state: stopped
+        state: started
```

Comment:
Why this change please?

Reply:
@jhutar We need to start the docker container to remove the images.
Before this change, the removal of images was raising the following error "Docker daemon not running"